patching/plugin-commands-patching/src/patchCommit.ts

```diff
@@ -127,6 +132,31 @@ export async function handler (opts: PatchCommitCommandOptions, params: string[]
   }) as Promise<undefined>
 }
 
+interface GetPatchContentContext {
+  patchedPkg: ParseWantedDependencyResult
+  patchedPkgDir: string
+  tmpName: string
+}
+
+type GetPatchContentOptions = Pick<PatchCommitCommandOptions, 'dir' | 'pnpmHomeDir' | 'storeDir'> & WritePackageOptions
+
+async function getPatchContent (ctx: GetPatchContentContext, opts: GetPatchContentOptions): Promise<string> {
+  const storeDir = await getStorePath({
+    pkgRoot: opts.dir,
+    storePath: opts.storeDir,
+    pnpmHomeDir: opts.pnpmHomeDir,
+  })
+  const srcDir = path.resolve(storeDir, 'tmp', 'patch-commit', ctx.tmpName)
```

Comment:
storeDir will be always an absolute path
```suggestion
  const srcDir = path.join(storeDir, 'tmp', 'patch-commit', ctx.tmpName)
```